lesson9/src/phonebook/PhoneBook.java

```diff
@@ -0,0 +1,33 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhoneBook {
+    private ArrayList<Record> records = new ArrayList<Record>();
+    public PhoneBook() {
```

Comment:
why did you need to declare this constructor? maybe I am mistaken but I see only one constructor and it is the default one as we discussed in one of the classes it will be created automatically